js_modules/dagster-ui/packages/ui-core/src/workspace/WorkspaceQueries.tsx

```diff
@@ -0,0 +1,138 @@
+import {gql} from '@apollo/client';
+
+import {REPOSITORY_INFO_FRAGMENT} from './RepositoryInformation';
+import {PYTHON_ERROR_FRAGMENT} from '../app/PythonErrorFragment';
+import {SENSOR_SWITCH_FRAGMENT} from '../sensors/SensorSwitch';
+
+export const LOCATION_WORKSPACE_QUERY = gql`
+  query LocationWorkspaceQuery($name: String!) {
+    workspaceLocationEntryOrError(name: $name) {
+      ...WorkspaceLocationNode
```

Comment:
should fragment for this root field erroring too to be safe

Reply:
We do check for the PythonError case but don't currently do anything with it (existing behavior). I think the real error that we do show is the loadOrError field